iroh/tests/cli.rs

```diff
@@ -244,7 +244,7 @@ fn cli_provide_tree_resume() -> Result<()> {
         let get_output = get.unchecked().run()?;
         assert!(get_output.status.success());
         let matches = explicit_matches(match_get_stderr(get_output.stderr)?);
-        assert_eq!(matches, vec!["112.84 KiB"]);
+        assert_eq!(matches, vec!["112.88 KiB"]);
```

Comment:
how come this is larger in total now?

Reply:
You got an additional indirection. Instead of having n links, you have n+1 links. That gives you an extra 32 bytes, or 0.032 kilobytes. I guess it gets rounded up...